charts/seq/templates/deployment.yaml

```diff
@@ -39,9 +39,18 @@ spec:
             - name: "SEQ_FIRSTRUN_ADMINUSERNAME"
               value: "{{ .Values.firstRunAdminUsername }}"
 {{- end }}
-{{- if .Values.firstRunAdminPassword }}
+{{- if or .Values.firstRunAdminPassword .Values.firstRunAdminPasswordSecret.name }}
             - name: "SEQ_FIRSTRUN_ADMINPASSWORD"
+{{- if or .Values.firstRunAdminPassword
```

Comment:
```suggestion
{{- if .Values.firstRunAdminPassword }}
```

Reply:
Thanks for catching my copy paste error.